### PR TITLE
Fix: rephrase index error message to not use technical term

### PR DESCRIPTION
**Pull request by aryagiat**
https://github.com/AY2122S1-CS2103T-F12-1/tp/issues/177

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -172,7 +172,7 @@ Deletes an existing question or note from the question bank or note list.
 Format: `delete question QUESTION_INDEX` OR `delete note NOTE_INDEX`
 
 * Deletes the question or note with the specified `QUESTION_INDEX` or `NOTE_INDEX`.
-* The `QUESTION_INDEX` or `NOTE_INDEX` refers to the index number shown in the displayed list. The index **must be a positive integer** 1, 2, 3, …​ 
+* The `QUESTION_INDEX` or `NOTE_INDEX` refers to the index number shown in the displayed list. The index **must be a positive integer** 1, 2, 3, …​ 2147483647
```
Comment (ryanpeh) [inline docs/UserGuide.md:175]: Perhaps specifying 1 to 2147483647 would be more appropriate here?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -255,12 +255,14 @@ Examples:
...
 #### 4.1.10. Start a Quiz: `quiz`
-Format: `quiz [lim/ LIMIT] [t/TAG]... [n/INDEX]`
+Format: `quiz [lim/ LIMIT] [t/TAG]... [n/INDEX...]`
 
 * If no parameters are passed, a quiz session will be created using all the questions in the question list.
 * LIMIT is a positive, non-zero integer that will limit the number of questions in the quiz.
   * If the total number of questions is less than the limit, it will just give all the questions.
 * TAG can be used to filter the quiz to only give questions with the tags specified, works with limit.
+* INDEX can be used to filter the quiz to only give questions with the specified question numbers. 
+  The index **must be a positive integer** 1, 2, 3, …​ 2147483647
```
Comment (ryanpeh) [inline docs/UserGuide.md:265]: Same as above.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -172,7 +172,7 @@ Deletes an existing question or note from the question bank or note list.
 Format: `delete question QUESTION_INDEX` OR `delete note NOTE_INDEX`
 
 * Deletes the question or note with the specified `QUESTION_INDEX` or `NOTE_INDEX`.
-* The `QUESTION_INDEX` or `NOTE_INDEX` refers to the index number shown in the displayed list. The index **must be a positive integer** 1, 2, 3, …​ 
+* The `QUESTION_INDEX` or `NOTE_INDEX` refers to the index number shown in the displayed list. The index **must be a positive integer** 1 - 2147483647
```
Comment (ryanpeh) [inline docs/UserGuide.md:175]: Sorry for nitpicking again but perhaps "The index **must be a positive integer from 1 - 2147483647**" would be clearer

File: docs/UserGuide.md [hunk 4]
```diff
@@ -255,12 +255,14 @@ Examples:
...
 #### 4.1.10. Start a Quiz: `quiz`
-Format: `quiz [lim/ LIMIT] [t/TAG]... [n/INDEX]`
+Format: `quiz [lim/ LIMIT] [t/TAG]... [n/INDEX...]`
 
 * If no parameters are passed, a quiz session will be created using all the questions in the question list.
 * LIMIT is a positive, non-zero integer that will limit the number of questions in the quiz.
   * If the total number of questions is less than the limit, it will just give all the questions.
 * TAG can be used to filter the quiz to only give questions with the tags specified, works with limit.
+* INDEX can be used to filter the quiz to only give questions with the specified question numbers. 
+  The index **must be a positive integer** 1 - 2147483647
```
Comment (ryanpeh) [inline docs/UserGuide.md:265]: Same as above

**Review by cedricongjh — APPROVED** [review]
LGTM

**Pull request merged**